### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [Terminus2049/Terminus2049.github.io#285]
The user @carlsonsantana validate your site template "https://terminus2049.github.io/" and found these accessibility errors:
* Page https://terminus2049.github.io/:
  * Issue *4dc7d272-673b-403b-94a8-a368afc4d013*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#main > form > input:nth-child(2)`
    * **Context**: `<input type="text" name="message" placeholder="留言（可选）" style="line-height:2; min-width:220px;">`
  * Issue *9205d528-867e-4966-b862-5bbd74c1d61d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#main > form > input:nth-child(2)`
    * **Context**: `<input type="text" name="message" placeholder="留言（可选）" style="line-height:2; min-width:220px;">`
  * Issue *6aef597e-2360-4bd6-9b34-a2556cc319ca*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputEmail.Name
    * **Message**: This emailinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#main > form > input:nth-child(1)`
    * **Context**: `<input type="email" name="email" placeholder="邮箱（请勿使用 qq 邮箱）" style="line-height:2; min-width:220px;" required="">`
  * Issue *d49830e9-18cb-445b-a08d-1ab9431af48d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#search-input`
    * **Context**: `<input type="text" id="search-input" placeholder="search..." style="margin: 0px 0px 20px 0px; height: 40px; width: 100%;">`
  * Issue *4ebca9f2-33de-47bb-81d3-f9dd9bced0a3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#search-input`
    * **Context**: `<input type="text" id="search-input" placeholder="search..." style="margin: 0px 0px 20px 0px; height: 40px; width: 100%;">`
  * Issue *c460f80b-e028-4760-a7da-db31bb257661*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
  <title>端点星 – 离银河中心最远的行...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](/validation/validations/572eff40-1207-4987-b811-c504e76f8d2e/).
